### PR TITLE
Add missing docstrings

### LINKED ISSUE #227: Missing docstrings in __init__
🐞 **Problem**

🖥 **Please paste the output of following commands**

* `conda info -a` (only if you have conda)
* `conda list` (only if you have conda)
* `pip freeze`

<details>

```bash
# Paste your output here:

```
</details>

* `python -c "import einsteinpy.testing; einsteinpy.testing.test()"`

<details>

```bash
# Paste your output here:

```
</details>

🎯 **Goal**

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by rtanmay**
Fixes #227 

@shreyasbapat 
@ritzvik

### CODE REVIEW

File: src/einsteinpy/metric/kerr.py [hunk 1]
```diff
@@ -18,7 +18,20 @@ class Kerr:
     """
 
     @u.quantity_input(time=u.s, M=u.kg)
-    def __init__(self, bl_coords, M, time):
+    def __init__(self, bl_coords, M, time=0 * u.s):
+        """
```
Comment (ritzvik) [inline src/einsteinpy/metric/kerr.py]: `__init__` method is not meant to be used normally, as we have made 2 other constructor methods for initialization

File: src/einsteinpy/metric/kerrnewman.py [hunk 2]
```diff
@@ -20,7 +20,24 @@ class KerrNewman:
     """
 
     @u.quantity_input(time=u.s, M=u.kg, Q=u.C)
-    def __init__(self, bl_coords, q, M, Q, time):
+    def __init__(self, bl_coords, q, M, Q, time=0 * u.s):
```
Comment (ritzvik) [inline src/einsteinpy/metric/kerrnewman.py]: Same here!

File: src/einsteinpy/metric/schwarzschild.py [hunk 3]
```diff
@@ -18,7 +18,20 @@ class Schwarzschild:
     """
 
     @u.quantity_input(time=u.s, M=u.kg)
-    def __init__(self, sph_coords, M, time):
+    def __init__(self, sph_coords, M, time=0 * u.s):
```
Comment (ritzvik) [inline src/einsteinpy/metric/schwarzschild.py]: And same here 😄

File: src/einsteinpy/metric/kerr.py (hunk 1 above)
Comment (rtanmay) [reply src/einsteinpy/metric/kerr.py]: okay I have uninitialised time from 0.
Is it okay now?

**Comment by rtanmay** [comment]
Changes done. Can you check now? @ritzvik 
Thanks

File: src/einsteinpy/metric/kerr.py (hunk 1 above)
Comment (ritzvik) [reply src/einsteinpy/metric/kerr.py]: What I meant is `__init__` don't need docstrings for all three classes under `metric`

Comment (rtanmay) [reply src/einsteinpy/metric/kerr.py]: But they are there in classes like velocity.py and core.py

Have removed them though as you said

Comment (ritzvik) [reply src/einsteinpy/metric/kerr.py]: They don't have explicit classmethods.
Thanks @rtanmay for the changes.
I would review it extensively in a while

**Comment by shreyasbapat** [comment]
I believe it's fine. More the merrier!

**Comment by ritzvik** [comment]
The incomplete docstrings shouldn't have passed our code review.
Anyways LGTM. @shreyasbapat 
Thanks @rtanmay

**Comment by shreyasbapat** [comment]
This all mess comes from #157 
We didn't had any docstrings for the classes prior to that!

**Pull request merged**